File: tests/cypress/cypress/elements/pages/utilities/cache_manager.rb
module CacheManager

  extend Capybara::DSL
  extend RSpec::Expectations
  extend RSpec::Matchers

  attr_accessor :button

  self.visit
    click_link 'Developer'
    click_link 'Utilities'
    click_link 'Cache Manager'
    no_php_js_errors

    @button = page.find('div.form-btns input.btn[type="submit"]');
  }

  self.button
    @button
  }

}

File: tests/cypress/cypress/elements/pages/utilities/CacheManager.js
import ControlPanel from '../ControlPanel'

class CacheManager extends ControlPanel {
  constructor() {
    super()
  }

  load() {
    this.open_dev_menu()
    this.get('main_menu').find('a:contains("Utilities")').click()
    this.get('wrap').find('a:contains("Cache Manager")').click()
  }


}
export default CacheManager;